src/components/app/app.tsx

```diff
@@ -1,5 +1,5 @@
 import { AppRouterProvider } from "src/router";
-import { AppAPIProvider } from "../api";
+import { AppAPIProvider } from "../../graphql";
```

Comment:
The 'vite-tsconfig-paths' library has already been added to the project, so you can replace paths like '../../graphql' with just 'src/graphql'.The Base URL is the root of the project. Maybe it would even be better to change it to the 'src' folder to just refer to folders inside "src" as "graphql" and so on. What do you all think?